mod_form.php

```diff
@@ -107,30 +113,23 @@ public function definition() {
         $mform->addElement('hidden', 'course', $id);
         $mform->addElement('header', 'general', get_string('general', 'form'));
         $mform->addElement('static', 'overview', get_string('overview', 'accredible'),
-            get_string('activitydescription', 'accredible'));
+            get_string('activitygroupdescription', 'accredible', $dashboardurl));
         if ($alreadyexists) {
             $mform->addElement('static', 'additionalactivitiesone', '', get_string('additionalactivitiesone', 'accredible'));
         }
-        $mform->addElement('text', 'name', get_string('activityname', 'accredible'), array('style' => 'width: 399px'));
-        $mform->addRule('name', null, 'required', null, 'client');
-        $mform->setType('name', PARAM_TEXT);
-        $mform->setDefault('name', $course->fullname);
```

Comment:
I think we should not remove the `activityname` field in case instructors want to display a different name such as `Certificate` to the student rather than the group name in Accredible.

Reply:
It makes sense, I have added the activity name again.

This is how the UI looks after this change
![Screenshot 2022-04-05 at 11 03 29](https://user-images.githubusercontent.com/91325424/161721611-597c0f3d-8255-4a73-afd7-ea0dfdc8ddf2.png)